test/src/e2e/chain.test.ts

```diff
@@ -252,42 +242,21 @@ describe("chain", function() {
 
     it("getNumberOfShards", async function() {
         expect(
-            await node.sdk.rpc.sendRpcRequest("chain_getNumberOfShards", [null])
-        ).to.equal(1);
+            await node.rpc.chain.getNumberOfShards()).to.equal(1);
 
         expect(
-            await node.sdk.rpc.sendRpcRequest("chain_getNumberOfShards", [0])
+            await node.rpc.chain.getNumberOfShards({blockNumber: 0})
         ).to.equal(1);
     });
 
     it("getShardRoot", async function() {
-        await node.sdk.rpc
-            .sendRpcRequest("chain_getShardRoot", [0, null])
-            .then(result => {
-                expect(result).not.to.be.null;
-                H256.ensure(result);
-            });
-
-        await node.sdk.rpc
-            .sendRpcRequest("chain_getShardRoot", [0, 0])
-            .then(result => {
-                expect(result).not.to.be.null;
-                H256.ensure(result);
-            });
-
-        await node.sdk.rpc
-            .sendRpcRequest("chain_getShardRoot", [10000, null])
-            .then(result => {
-                expect(result).to.be.null;
-            });
+        expect(await node.rpc.chain.getShardOwners({shardId: 0, blockNumber: null})).not.to.be.null;
```

Comment:
What this line is doing is different from the previous behavior. You should call `H256.ensure` for the return value of `getShardOwners`.